### ISSUE TITLE
Credit field unavailable to regular user - Missing field in blog post submission form

### ISSUE THREAD

**Issue by lucas-menegazzo** [lucas-menegazzo/ml-linker#5]
O campo de crédito ainda não está disponível para o usuário comum.

## Images

![Interface gráfica do usuário, Texto, Email

O conteúdo gerado por IA pode estar incorreto.](https://raw.githubusercontent.com/lucas-menegazzo/ml-linker/main/issue-images/1767905550924-Interface_gr_fica_do_usu_rio__Texto__Email__O_conte_do_gerado_por_IA_pode_estar_incorreto..png)
